modules/docs/Makefile

```diff
@@ -0,0 +1,21 @@
+GOMPLATE_VERSION ?= v2.2.0
+GOMPLATE_REPO ?= hairyhenderson/gomplate
+GOMPLATE ?= gomplate
+
+## Install dependencies
+docs\:deps:
+	@make github:download-public-release \
+		REPO=$(GOMPLATE_REPO) \
+		FILE=$(GOMPLATE)_$(OS)-amd64 \
+		VERSION=$(GOMPLATE_VERSION) \
+		OUTPUT=/usr/local/bin/$(GOMPLATE) > /dev/null
+	@chmod +x /usr/local/bin/$(GOMPLATE)
+
+
+
+.PHONY : docs\:build
+## Create $OUT file by building it from $IN template file
+docs\:build:
+	@BUILD_HARNESS_PATH=$(BUILD_HARNESS_PATH) \
```

Comment:
This is already exported, so should need to be passed

Reply:
When I remove pass $BUILD_HARNESS_PATH
I got this error

```
2017/12/04 15:34:19 Can't read /home/goruha/projects/cloudposse/build-harness/: &os.PathError{Op:"read", Path:"/home/goruha/projects/cloudposse/build-harness/", Err:0x15}
README.md doc generated
```